packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -3529,6 +3529,51 @@ describeWithDOM('mount', () => {
       const formUp = input.parents('form');
       expect(formUp).to.have.lengthOf(1);
     });
+
+    it('works when called sequentially on two sibling nodes', () => {
+      class Test extends React.Component {
+        render() {
+          return (
+            <div>
+              <div className="a">
+                <div>A child</div>
+              </div>
+              <div className="b">
+                <div>B child</div>
+              </div>
+            </div>
+          );
+        }
+      }
+
+      const wrapper = mount(<Test />);
+
+      const aChild = wrapper.find({ children: 'A child' });
+      expect(aChild.debug()).to.equal(`<div>
+  A child
+</div>`);
+      expect(aChild).to.have.lengthOf(1);
+
+      const bChild = wrapper.find({ children: 'B child' });
+      expect(bChild.debug()).to.equal(`<div>
+  B child
+</div>`);
+      expect(bChild).to.have.lengthOf(1);
+
+      /*
+      const bChildParents = bChild.parents('.b');
+      expect(bChildParents.debug(`<div className="b">
+  <div>B child</div>
+</div>`));
+      expect(bChildParents).to.have.lengthOf(1);
+      */
+
+      const aChildParents = aChild.parents('.b');
```

Comment:
@ljharb I believe this should this be `const aChildParents = aChild.parents('.a');`? (which fixes this test)

Reply:
thanks, i'll fix that